cli/azd/cmd/auth_login.go

```diff
@@ -354,8 +377,31 @@ func (la *loginAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 		}
 	}
 
-	la.console.Message(ctx, "Logged in to Azure.")
-	return nil, nil
+	const cLoginSuccessMessage = "Logged in to Azure"
+	details, err := la.authManager.LogInDetails(ctx)
+
+	// error getting user account, successful log in
+	if err != nil {
+		log.Printf("error: getting signed in account: %v", err)
+		la.console.Message(ctx, cLoginSuccessMessage+".")
+		return nil, nil
+	}
+
+	switch details.LoginType {
+	case auth.EmailLoginType:
+		fmt.Fprintf(la.console.Handles().Stdout, "%s as %s.", cLoginSuccessMessage, output.WithBold("%s", details.Account))
+		return nil, nil
+	case auth.ClientIdLoginType:
+		fmt.Fprintf(
+			la.console.Handles().Stdout,
+			"%s as (%s).",
+			cLoginSuccessMessage,
+			output.WithGrayFormat("%s", details.Account))
+		return nil, nil
+	default:
+		la.console.Message(ctx, cLoginSuccessMessage+".")
+		return nil, nil
+	}
```

Comment:
Is this code the same as the one above?  

Here's a PR with how you can use UX items to describe and re-use output in azd: https://github.com/john0isaac/azure-dev/pull/2

Ux items let us keep track of indentation and have some single source to define the output elements that can be used from different packages.

Reply:
kinda the difference is that this only triggers on successful login while the above added lines are for `--check-status` which have to outputs either logged in or not logged in.